pennylane/math/tensorbox.py

```diff
@@ -104,28 +104,35 @@ def __new__(cls, tensor):
             return super(TensorBox, cls).__new__(cls)
 
         namespace = tensor.__class__.__module__.split(".")[0]

```

Comment:
```
>>> from jax import numpy as np
>>> x = np.array([0.1, 0.2])
>>> x
DeviceArray([0.1, 0.2], dtype=float32)
>>> x.__class__.__module__.split(".")[0]
'jax'
```

😌 

This is the only 'clean' way we could work out for determining the ML framework from the array/tensor object without importing the framework itself, so I'm glad it continues to work with JAX! All it takes is for a single framework to use some weird namespacing though for this to fail.